windows/d2-installer.nsi

```diff
@@ -295,6 +295,7 @@ SectionGroup /e "D2"
 
   Section "Add to PATH" AddD2ToPath
     ${EnvVarUpdate} $0 "PATH" "A" "HKLM" "$INSTDIR\dmd2\windows\bin"
+    ${EnvVarUpdate} $0 "PATH" "A" "HKLM" "$INSTDIR\dmd2\windows\bin64"
```

Comment:
@CyberShadow This was the only place where the path wasn't set to bin64 although I haven't found a way to do it depending on the arch.

Reply:
Although I have no idea why this [1] isn't working. I haven't done windows stuff since forever.

[1] https://github.com/dlang/installer/pull/484/files#diff-179a67f45668adad5ec8526432aeda299d6c08b4df2bb5349f37489fda1674d4R277